JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -0,0 +1,93 @@
+https://www.theworldavatar.com/kg/ontotimeseries/hasTimeSeries,Object Property,,,https://saref.etsi.org/core/State,https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,,,https://www.theworldavatar.com/kg/ontodevice/,has timeseries
+https://www.theworldavatar.com/kg/ontotimeseries/hasTimeSeries,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,,,https://www.theworldavatar.com/kg/ontodevice/,has timeseries
+https://www.theworldavatar.com/kg/ontotimeseries/hasTimeSeries,Object Property,,,http://www.opengis.net/ont/sf#Point,https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,,,https://www.theworldavatar.com/kg/ontodevice/,has timeseries
+isAttachedTo,Object Property,,,Device,https://w3id.org/bot#Element,,A relationship that describes a device being attached to another physical element or object.,https://www.theworldavatar.com/kg/ontodevice/,is attached to
+isConnectedTo,Object Property,,,Device,Device,,A relationship that describes a device being connected to another device where the operation of one device will affect the other.,https://www.theworldavatar.com/kg/ontodevice/,is connected to
+sendsSignalTo,Object Property,,,Device,Device,,A relationship that describes a device sending a signal to another device.,https://www.theworldavatar.com/kg/ontodevice/,sends signal to
+hasGeoLocation,Object Property,,,Device,http://www.opengis.net/ont/sf#Point,,A relationship that describes a device's location.,https://www.theworldavatar.com/kg/ontodevice/,has geo location
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasValue,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has value
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has unit
+http://www.opengis.net/ont/geosparql#asWKT,Data Property,,,http://www.opengis.net/ont/sf#Geometry,http://www.opengis.net/ont/geosparql#wktLiteral,,,http://www.opengis.net/ont/geosparql#,as WKT
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasNumericalValue,Data Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,double,,A relationship that links a measure to a double value.,https://www.theworldavatar.com/kg/ontodevice/,has numerical value
+http://www.w3.org/2000/01/rdf-schema#label,Data Property,,,Device,String,,A relationship that links a device to a string label.,https://www.theworldavatar.com/kg/ontodevice/,label
```

Comment:
rdfs:label already has a very wide domain definition - there is no need to explicitly define it for a Device. Please remove this.